### ISSUE TITLE
Screenreader issue for read-only text fields

### ISSUE THREAD

**Issue by sswweber** [vaadin/vaadin-text-field-flow#298]
When using a screenreader for read-only text fields, the screenreader reads a bullet point to seperate label from textfield instead of the usual asterisk '*'. These bullet points are also read to users which have no meaning to them.
To mark the read-only attribute, vaadin uses an additional HTML5-Attribute '<aria-readonly>' which is never set to true. Therefore the attribute cannot be interpreted correctly by the screenreader.
